build/Build.cmd

```diff
@@ -142,16 +168,16 @@ xcopy /e /y "%CMDHOME%\..\scripts"  "%SPARKCLR_HOME%\scripts\"
 pushd "%CMDHOME%"
 if not exist ".\target" (mkdir .\target)
 
+set ProjectVersion=1.6.0-preview-3
```

Comment:
It looks like the merge between my private branch and master has problems. this line is just for my test propose. Actually, I have removed the line before commit. Not sure why it came back.

Let me do a clean branch again.